README.md

```diff
@@ -243,6 +243,20 @@ flavor:
 * `INSTANCE_MYSQL_URL` If using an external mysql database, set its url here
 * `INSTANCE_MONGO_URL` If using an external mongo database, set its url here
 
+### External SMTP service settings
+
+If you want to use an external SMTP service for sending email from app servers,
+set the following configuration variables:
+
+* `INSTANCE_SMTP_RELAY_HOST` External SMTP host
+* `INSTANCE_SMTP_RELAY_PORT` External SMTP port
+* `INSTANCE_SMTP_RELAY_USERNAME`: External SMTP provider username
+* `INSTANCE_SMTP_RELAY_PASSWORD`: External SMTP provider password
+* `INSTANCE_SMTP_RELAY_SENDER_DOMAIN`: When using external SMTP provider, email
```

Comment:
@itsjeyd lol, I have no idea how I managed to start without colons and then start adding them half way through the list. I added colon to all settings, it really makes the text more readable.